.editorconfig

```diff
@@ -0,0 +1,14 @@
+# https://editorconfig.org/
+
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+
+[*.js]
+indent_style = tab
+
+[*.{json,nix,purs,yaml,yml}]
+indent_style = space
+indent_size = 2
```

Comment:
We should get rid of this file from this PR, and perhaps make it part of its own PR. That said I don't think I want `.editorconfig` on this repo yet ...

Reply:
I can drop this, but I wanted/needed it for my editor situation.